README.md

```diff
@@ -89,7 +89,7 @@ to install version 0.13.13 of `terratest_log_parser`:
 ```bash
 # This example assumes a linux 64bit machine
 # Use curl to download the binary
-curl -o terratest_log_parser https://github.com/gruntwork-io/terratest/releases/download/v0.13.13/terratest_log_parser_linux_amd64
+curl -Lo terratest_log_parser https://github.com/gruntwork-io/terratest/releases/download/v0.13.13/terratest_log_parser_linux_amd64
```

Comment:
We typically use:

```
curl --location --silent --fail --show-error <URL>
```

That seems to be a good mix of handling redirects and errors and showing error messages but not other noise.